src/serlo-editor/plugins/text/components/math-element.tsx

```diff
@@ -149,50 +185,28 @@ export function MathElement({
   function transformOutOfElement({
```

Comment:
If you defined a new handler (or two) for strictly using Esc and close button to close the modal, you could avoid the long comment down there, a third argument to this function, and instead of one clunked function, you would have multiple smaller ones which do simple things that are easy to explain in one function name.

This `transformOutOfElement` function was too big already before this PR, as it handles left arrow key, right arrow key, backspace key, and delete key. And all of that for not one, but two different components. 😅  So making it do even more seems like a bad idea to me.